integrations/cdi/narayana-jta-cdi/src/main/java/io/helidon/integrations/narayana/jta/cdi/NarayanaTransactionSynchronizationRegistry.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.narayana.jta.cdi;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import com.arjuna.ats.jta.common.JTAEnvironmentBean;
+
+/**
+ * A {@link DelegatingTransactionSynchronizationRegistry} in
+ * {@linkplain ApplicationScoped application scope} that uses the
+ * return value that results from invoking the {@link
+ * JTAEnvironmentBean#getTransactionSynchronizationRegistry()} method
+ * as its backing implementation.
+ *
+ * @author <a href="mailto:laird.nelson@oracle.com">Laird Nelson</a>
+ *
+ * @see JTAEnvironmentBean#getTransactionSynchronizationRegistry()
+ */
+@ApplicationScoped
+class NarayanaTransactionSynchronizationRegistry extends DelegatingTransactionSynchronizationRegistry {
+
+  /**
```

Comment:
This file seems to be indented with 2 spaces instead of 4.

Reply:
Done.